### DISCUSSION TITLE
Add new Photovoltaic-Modules

### DISCUSSION THREAD

**Discussion by jh9604** [SESMG/SESMG#137] [Q&A]
I wanted to add new Photovoltaic-Modules and Inverter and saw that the calculation and the database in based on Sandia Model. After reading some infos about Sandia Model I found out that it wouldn't been possible to add new Moduldatas, because Sandial Model works with lot's of measurement parameters that are not given in normal Datasheets.

Is it possible to use the CEC_database in SESMG to work with common parameters that are given in common Datasheets? Are some changes in Code necessary? Maybe someone else had a similar plan before and can help me with it.

_I'm really new in working with SESMG, so sorry if it's a stupid question._

**Comment by jtock** [comment]
Dear @jh9604,

thank you for your question. First of all: it is not a stupid one!

The SESMG is using the oemof feedinlib as a subpackage. We are using their functions to implement the pv modul data into the SESMG.
The CEC database can not be used by the feedinlib. (See: https://github.com/oemof/feedinlib/issues/10)
If you can figure out if there are databases that can be combined with the feedinlib and deliver the informations you need, we can help you to implement the database into the SESMG. It is also possible that there is a way to use the CEC database with the feedinlib, which can then be implemented into the SESMG.

Greeting
SESMG Dev-Team
